README.md

```diff
@@ -1,2 +1,114 @@
-# strigo-github-exercises-examples
-Examples for markdown exercises that may be used in Strigo training events
+# Strigo GitHub Class Exercises examples
+
+## Quick start
+
+1. Create `.strigo/config.yml` file in your exercises repository
+1. Configure exercises in `.strigo/config`
+
+   ```yaml
+   ---
+   classes:
+     - exercises:
+         # `file` is relative to .strigo directory
+         - file: "../Exercise.md"
+           title: "Title!"
+   ```
+
+1. In Strigo Class editor, fetch Exercises for a desired Class
```

Comment:
I'd use "Edit your class template in Strigo and chose to fetch exercises from GitHub".

Reply:
We don't use the term "template" in Lounge, so maybe just "Edit your class"?

Reply:
We use class template when talking to customers all the time.. but I agree that it's inconsistent. "class" is good.